src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -755,9 +848,9 @@ private SortedSet<String> createSampleNameInclusionList(Map<String, VCFHeader> v
         final SortedSet<String> vcfSamples = VcfUtils.getSortedSampleSet(vcfHeaders, GATKVariantContextUtils.GenotypeMergeType.REQUIRE_UNIQUE);
         final Collection<String> samplesFromExpressions = Utils.filterCollectionByExpressions(vcfSamples, sampleExpressions, false);
 
-        // first, find any samples that were listed on the command line but which don't exist in in the header
+        // first, find any samples that were listed on the command line but don't exist in the header
         final Set<String> samplesNotInHeader = new LinkedHashSet<>(samplesFromExpressions.size()+sampleNames.size());
-        samplesNotInHeader.addAll(samplesFromExpressions);
+        // samplesNotInHeader.addAll(samplesFromExpressions); // tsato: samplesFromExpressions is a subset of vcfsamples, so this should be unnecessary
```

Comment:
Yes, agree that it seems redundant -- I think you can remove this line.

Reply:
Done